linked_list/interface.py

```diff
@@ -29,7 +29,7 @@ def __eq__(self, other):
         raise NotImplementedError()
 
     def __ne__(self, other):
-        raise NotImplementedError()
+        return not self == other
```

Comment:
If all your other code for the list is in the implementation file why are you choosing to place logic in here? (Also the comments at the beginning of the file state to not implement any of the methods in it)